src/Tokenizer/Tokens.php

```diff
@@ -1127,19 +1127,21 @@ public function clearRange(int $indexStart, int $indexEnd): void
      */
     public function isMonolithicPhp(): bool
     {
-        if (0 === $this->count()) {
-            return false;
-        }
+        if ((1 === $this->countTokenKind(T_OPEN_TAG) && 0 === $this->countTokenKind(T_OPEN_TAG_WITH_ECHO))
+            || (0 === $this->countTokenKind(T_OPEN_TAG) && 1 === $this->countTokenKind(T_OPEN_TAG_WITH_ECHO))
+        ) {
+            if ($this->countTokenKind(T_INLINE_HTML) > 1) {
+                return false;
+            }
 
-        if ($this->countTokenKind(T_INLINE_HTML) > 1) {
-            return false;
-        }
+            if (1 === $this->countTokenKind(T_INLINE_HTML)) {
+                return Preg::match('/^#!.+$/', $this[0]->getContent());
+            }
 
-        if (1 === $this->countTokenKind(T_INLINE_HTML)) {
-            return Preg::match('/^#!.+$/', $this[0]->getContent());
+            return true;
         }
 
-        return 1 === ($this->countTokenKind(T_OPEN_TAG) + $this->countTokenKind(T_OPEN_TAG_WITH_ECHO));
+        return false;
```

Comment:
Why do you rewrite this method? The behavior seems to be the same but in my opinion previous version was easier to read.